src/interactivity/Interactivity.js

```diff
@@ -200,14 +217,19 @@ export default class Interactivity {
             });
         }
 
-        this._prevHoverFeatures = featureEvent.features;
+        this._prevHoverFeatures = currentFeatures;
 
-        // Launch hover event
-        this._fireEvent('featureHover', featureEvent);
+        // If the event comes from a real mouse move, trigger always (because coordinates and position have changed)
+        // If the event comes from an animated event, trigger only when features have changed (because position is the same)
+        if (!force || (force && (featuresLeft.length || featuresEntered.length))) {
```

Comment:
I would change the name of `force` since it gives me the opposite meaning to me. I think `virtual` or `emulated` is better.